docs/index.md

```diff
@@ -43,234 +43,73 @@ theme:
-`Vectors`, `Shapes`, and `Surface`, each corresponding to a different data type,
-visualization, and interactivity. You can add multiple layers of different types
-into the viewer and then start working with them, adjusting their properties.
-
-All our layer types support n-dimensional data and the viewer provides the
-ability to quickly browse and visualize either 2D or 3D slices of the data.
-
-**napari** also supports bidirectional communication between the viewer and the
-Python kernel, which is especially useful when launching from jupyter notebooks
-or when using our built-in console. Using the console allows you to
-interactively load and save data from the viewer and control all the features of
-the viewer programmatically.
-
-You can extend **napari** using custom shortcuts, key bindings, and mouse
-functions.
-
-## Tutorials
+Napari is a Python library for n-dimensional image visualisation, annotation,
+and analysis. With napari you can:
+- **view and explore** 2D, 3D, and higher-dimensional arrays on a canvas;
+- **overlay** derived data such as *points*, *polygons*, *segmentations*, and
+  more;
+- **annotate** and **edit** derived datasets, using standard data structures
+  such as NumPy or Zarr arrays, allowing you to
+- **seamlessly weave** exploration, computation, and annotation together in
+  imaging data analysis.
 
-For more details on how to use `napari` checkout our [tutorials](./tutorials/index).
-These are still a work in progress, but we'll be updating them regularly.
+::::{grid}
 
-## Mission, values, and roadmap
+:::{grid-item-card} Examples gallery
+:link: gallery
+:link-type: ref
 
-For more information about our plans for `napari` you can read our [mission and
-values statement](./community/mission_and_values.md), which includes more
-details on our vision for supporting a plugin ecosystem around napari.
+See some of the things napari can do.
+:::
 
-## Contributing
+:::{grid-item-card} Installation
+:link: installation
+:link-type: ref
 
-Contributions are encouraged! Please read our [contributing
-guide](./developers/contributing.md) to get started. Given that we're in an
-early stage, you may want to reach out on our [Github
-Issues](https://github.com/napari/napari/issues) before jumping in.
+How to install napari.
+:::
 
-## Code of conduct
+:::{grid-item-card} Getting started
+:link: getting_started
+:link-type: ref
 
-`napari` has a [Code of Conduct](./community/code_of_conduct) that should be
-honored by everyone who participates in the `napari` community.
+Get started with napari.
+:::
 
-## Governance
+::::
 
-You can learn more about how the `napari` project is organized and managed from
-our [governance model](./community/governance), which includes information
-about, and ways to contact, the
-[@napari/steering-council](https://github.com/orgs/napari/teams/steering-council)
-and [@napari/core-devs](https://github.com/orgs/napari/teams/core-devs).
+::::{grid}
```

Comment:
But You hardcode it here as three items in a grid so the observed behavior 2-1-2-1 is correct

Reply:
Yes that's what I meant by "but I think that's hard/impossible with the existing syntax..."